lib/srv/exec.go

```diff
@@ -258,11 +249,29 @@ func (e *localExec) transformSecureCopy() error {
 		teleportBin,
 		e.Ctx.ServerConn.RemoteAddr().String(),
 		e.Ctx.ServerConn.LocalAddr().String(),
-		strings.Join(args[1:], " "))
+		scpArgs,
+	)
 
 	return nil
 }
 
+func checkSCPAllowed(scx *ServerContext, command string) error {
```

Comment:
Should we rename this given that we don't actually use SCP anymore?

Reply:
We still support it server-side unfortunately :/